common/src/main/java/com/wynntils/core/net/UrlId.java

```diff
@@ -32,7 +32,7 @@ public enum UrlId {
     DATA_STATIC_ABILITIES("dataStaticAbilities"),
     DATA_STATIC_CAVE_INFO("dataStaticCaveInfo"),
     DATA_STATIC_CHARMS("dataStaticCharms"),
-    DATA_STATIC_COMBAT_LOCATIONS("dataStaticCombatLocations"),
+    DATA_STATIC_MAPDATA_COMBAT_LOCATIONS("dataStaticMapdataCombatLocations"),
```

Comment:
I'd like to avoid "mapdata" as a term other than when we are doing this transition, so we do not encode it in user-fronting code, or essentially in anything except the `mapdata` package. It kind of makes sense now, but when it is the only kind of map data we have it will sound wonky.

What about `dataStaticCombatMapFeatures` or `dataStaticCombatMapLocations`?

Reply:
I specifically include mapdata in the URLs because otherwise Static-Storage becomes quite hard to differentiate. I guess "MapFeatures" can work.

Reply:
MapFeatures is better than map data, so if that is okay, please go with it.